src/lib/util/cdrom.h

```diff
@@ -169,6 +172,16 @@ static inline uint32_t lba_to_msf(uint32_t lba)
 			((f / 10) <<  4) | ((f % 10) <<  0);
 }
 
+static inline uint32_t reverse32(uint32_t x)
+{
+	x = ((x & 0x55555555) <<  1) | ((x & 0xAAAAAAAA) >>  1);
+	x = ((x & 0x33333333) <<  2) | ((x & 0xCCCCCCCC) >>  2);
+	x = ((x & 0x0F0F0F0F) <<  4) | ((x & 0xF0F0F0F0) >>  4);
+	x = ((x & 0x00FF00FF) <<  8) | ((x & 0xFF00FF00) >>  8);
+	x = ((x & 0x0000FFFF) << 16) | ((x & 0xFFFF0000) >> 16);
+	return x;
+}
```

Comment:
There are helpers for doing this stuff in one of the headers in src/osd already.

Reply:
You’ll have to be more specific. I don’t know the helper you’re thinking of.